### PR TITLE
zot: cleanup logs

### PR DESCRIPTION
**Pull request by rchincha**
- add logs only in the error paths

### CODE REVIEW

File: zot/image_dest.go [hunk 1]
```diff
@@ -74,16 +74,17 @@ func (o *zotImageDest) PutBlob(ctx context.Context, stream io.Reader, inputInfo
 		}
 	}
 
-	fmt.Printf("calling startlayer\n")
 	// Do this as a chunked upload so we can calculate the digest, since
 	// caller is not giving it to us.
 	path, err := o.s.StartLayer()
-	fmt.Printf("callled startlayer and got %s %v\n", path, err)
 	if err != nil {
+		fmt.Errorf("called StartLayer and got path(%s) and error(%v)", path, err)
```
Comment (tych0) [inline zot/image_dest.go]: This looks like a no-op, as does the one below?

Comment (rchincha) [reply zot/image_dest.go]: The idea was to keep the logs in the error path (assuming zot could misbehave and we are able to debug)

Comment (tych0) [reply zot/image_dest.go]: Right, but fmt.Errorf() by itself does nothing; you need to assign the result somewhere or print it, right?

Comment (rchincha) [reply zot/image_dest.go]: ah, meant to use logrus, that is what copy/copy.go uses

**Pull request merged**